src/CMakeLists.txt

```diff
@@ -102,6 +114,9 @@ endif (MOAB_FOUND)
 set_source_files_properties(decay.cpp PROPERTIES COMPILE_FLAGS 
     "-O0")
 
+add_executable(alphad ${PROJECT_SOURCE_DIR}/src/ensdf_processing/ALPHAD/alphad.f)
+target_link_libraries(alphad pyne)
+
```

Comment:
Try `target_link_libraries(alphad LINK_PUBLIC pyne)`.  What does google and stackoverflow say about the problem?